packages/headless/src/features/generated-answer/generated-answer-insight-analytics-actions.ts

```diff
@@ -16,8 +16,26 @@ export type GeneratedAnswerFeedback =
   | 'notAccurate'
   | 'outOfDate'
   | 'harmful';
+
+export type GeneratedAnswerFeedbackOption = 'yes' | 'unknown' | 'no';
+
+export type GeneratedAnswerFeedbackV2 = {
+  helpful: boolean;
+  documented: GeneratedAnswerFeedbackOption;
+  correctTopic: GeneratedAnswerFeedbackOption;
+  hallucinationFree: GeneratedAnswerFeedbackOption;
+  readable: GeneratedAnswerFeedbackOption;
+  details?: string;
+  documentUrl?: string;
+};
```

Comment:
I think we should import these types from the `generated-answer-analytics-actions.ts` file instead of redeclaring them